File: TransCoder_tokenized_test_set_functions/java/CONVERT_STRING_BINARY_SEQUENCE.java
static void strToBinary ( String s ) {
  int n = s . length ( );
  for ( int i = 0;
  i < n;
  i ++ ) {
    int val = Integer . valueOf ( s . charAt ( i ) );
    String bin = "";
    while ( val > 0 ) {
      if ( val % 2 == 1 ) {
        bin += '1';
      }
      else bin += '0';
      val /= 2;
    }
    bin = reverse ( bin );
    System . out . print ( bin + " " );
  }
}

File: TransCoder_tokenized_test_set_functions/cpp/CONVERT_STRING_BINARY_SEQUENCE.cpp
void strToBinary ( string s ) {
  int n = s . length ( );
  for ( int i = 0;
  i <= n;
  i ++ ) {
    int val = int ( s [ i ] );
    string bin = "";
    while ( val > 0 ) {
      ( val % 2 ) ? bin . push_back ( '1' ) : bin . push_back ( '0' );
      val /= 2;
    }
    reverse ( bin . begin ( ), bin . end ( ) );
    cout << bin << " ";
  }
}